examples/experimental/dagster-airlift/README.md

```diff
@@ -130,20 +130,22 @@ The first and third tasks involve a single table each. We can manually construct
 provided to the `defs` argument to `build_defs_from_airflow_instance`.
 
 ```python
+from dagster import AssetSpec
+
+from dagster_airlift.core import build_defs_from_airflow_instance, dag_defs, task_defs
+
 defs = build_defs_from_airflow_instance(
     airflow_instance=airflow_instance,
     defs=dag_defs(
         "rebuild_customers_list",
         task_defs(
             "load_raw_customers",
-            AssetSpec(key=AssetKey(["raw_data", "raw_customers"]))
```

Comment:
Why is the "raw_data" piece even necessary here? Is this an artifact of dbt?

Reply:
Not sure. I wasn't really examining the core content here, just cleaning up what is there already.